### ISSUE TITLE
Pin Calendar Tab To Top Bar

### ISSUE THREAD

**Issue by valiantgenomics** [davish/obsidian-full-calendar#177]
Just a quick and humble quality of life proposition.

Would it be possible to "pin" the calendar to the top bar, just like you can pin other notes to the top bar, by right-clicking and selecting "pin"?

![image](https://user-images.githubusercontent.com/111633318/201181632-764b1aef-91ef-4e06-a4bb-8a9d801db9a1.png)

Unless I'm missing something, but I cannot see the "Pin" option when I right-click on the Calendar tab.

This will enable us to keep our calendar always at the forefront and not only switch to it faster than pressing the hotkey, but also it kind of serves as a reminder to check your calendar more often and plan accordingly.

Thanks!!!

**Comment by valiantgenomics** [comment]
A thing I noticed during the past few days of using the calendar. 

Since I keep it always open and essentially pinned to my tab, without actually pinning it, the current date-highlight doesn't change. 

For example, if it's already the next day, the highlight still shows the previous day. So you have to manually restart the calendar to get it to update. Would be awesome if the date-highlight changes automatically, and we don't have to restart the calendar.

Thanks!!!!

**Comment by MiracleXYZ** [comment]
You can actually use the Command Palette and pin the tab, but notes would still be opened in this tab, which is a bit annoying.

**Comment by Alejandro-Stolz** [comment]
> 

For me they don't, they open to the side like any other pinned tab.

**Comment by Alejandro-Stolz** [comment]
> 



> I used the command palette and could pin and notes opened in new tabs. But rather than the extra step how about adding an option to always pin when opened.

Yes, that would also work. For now, I've found a workaround. You can create a macro with quickadd, that opens the calendar and toggles the pin status. And with commander you can add it to the ribbon as an icon.
This works, but I really would prefer normal tab behaviour for the plugin so that it has the note menu that allows to pin it, etc.

**Comment by valiantgenomics** [comment]
Found a workaround for this issue, that essentially solves it completely. 

Open your full calendar. Then open a new tab separately.

Right-click on the new tab, and link it with the calendar tab.

Then right-click on the new tab again and pin it.

Then unlink it with the calendar tab.

Finally, unpin the new tab and close it.

Now your calendar tab should've inherited the Pin command from the new tab, and it will stay pinned in perpetuity, until you update the Full Calendar plugin. Then you need to re-do the process again.

And no, new tabs won't open in its place. The calendar tab will fully behave like a pinned tab.

Cheers.
